### DISCUSSION TITLE
API Resources

### DISCUSSION THREAD

**Discussion by smokills** [filamentphp/filament#1349] [Help] [answered]
There is any change to build a CRUD for a third party API Service? For example, I have an external service that esposes a CRUD via REST APIs, is possible to generate a page that will build tables and forms to handle that APIs?

**Comment by danharrin** [comment, marked as answer]
If you want to use resources, it'll need to be though an Eloquent model. So maybe you'd want to go through [Sushi](https://github.com/calebporzio/sushi), which could handle data retrieval? And have custom `save()` and `delete()` methods for your model that could handle creation / updates / deletion via the API?

**Reply by smokills** [reply]
That rocks 🤘 ! Or maybe I can got inspired by that, and create something like Sushi, but that will make API requests. Thanks a LOT!

**Reply by danharrin** [reply]
Awesome! Looking forward to seeing it 👍

**Reply by Krato** [reply]
> That rocks 🤘 ! Or maybe I can got inspired by that, and create something like Sushi, but that will make API requests. Thanks a LOT!

We need to implement something like this. But we don't want to read all items every time. How you approach? Thanks
